airbyte-webapp/src/packages/cloud/views/auth/SignupPage/components/SignupForm.tsx

```diff
@@ -115,6 +191,30 @@ export const PasswordField: React.FC<{ label?: React.ReactNode }> = ({ label })
   );
 };
 
+export const RHFPasswordField: React.FC = () => {
+  const { formatMessage } = useIntl();
+  const { control } = useFormContext<SingupFormValues>();
+
+  return (
+    <Controller
+      name="password"
+      control={control}
```

Comment:
This is more react-hook-form's issue but instead of having to use the context and pass it to the Controller, `<Controller<SignupFormValues>>` would be a better pattern.